tools/azure_controller.sh

```diff
@@ -184,7 +184,7 @@ elif [ $STAGE != "$STAGE_CLEANUP" ]; then
     fi
 
 
-    TEST="$STAGE" "./tools/travis_watchdog.sh" 300
+    TEST="$STAGE" "./tools/travis_watchdog.sh" 900
```

Comment:
might as well remove the explicit argument if we will always keep it in sync with the watchdog anyway

Reply:
Good point!
Let's see where this ends up after my "clean up CI scripts" PR :)